appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/deployment/node/EjbNode.java

```diff
@@ -48,25 +48,7 @@
 import com.sun.enterprise.deployment.MessageDestinationReferenceDescriptor;
 import com.sun.enterprise.deployment.RoleReference;
 import com.sun.enterprise.deployment.RunAsIdentityDescriptor;
-import com.sun.enterprise.deployment.node.AdministeredObjectDefinitionNode;
-import com.sun.enterprise.deployment.node.ConnectionFactoryDefinitionNode;
-import com.sun.enterprise.deployment.node.DataSourceDefinitionNode;
-import com.sun.enterprise.deployment.node.JMSConnectionFactoryDefinitionNode;
-import com.sun.enterprise.deployment.node.JMSDestinationDefinitionNode;
-import com.sun.enterprise.deployment.node.MailSessionNode;
-import com.sun.enterprise.deployment.node.DisplayableComponentNode;
-import com.sun.enterprise.deployment.node.EjbLocalReferenceNode;
-import com.sun.enterprise.deployment.node.EjbReferenceNode;
-import com.sun.enterprise.deployment.node.EntityManagerFactoryReferenceNode;
-import com.sun.enterprise.deployment.node.EntityManagerReferenceNode;
-import com.sun.enterprise.deployment.node.EnvEntryNode;
-import com.sun.enterprise.deployment.node.JndiEnvRefNode;
-import com.sun.enterprise.deployment.node.MessageDestinationRefNode;
-import com.sun.enterprise.deployment.node.ResourceEnvRefNode;
-import com.sun.enterprise.deployment.node.ResourceRefNode;
-import com.sun.enterprise.deployment.node.SecurityRoleRefNode;
-import com.sun.enterprise.deployment.node.XMLElement;
-import com.sun.enterprise.deployment.node.ManagedExecutorDefinitionNode;
+import com.sun.enterprise.deployment.node.*;
```

Comment:
Star imports are the way to future incompatibility. Things like XMLElement can easily become a problem.

Reply:
already done